sig-scalability/charter.md

```diff
@@ -0,0 +1,110 @@
+# SIG Scalability Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README]
+and uses the Roles and Organization Management outlined in [sig-governance].
+
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+
+## Scope
+
+SIG Scalability primary responsibilities are to define scalability goals for
+Kubernetes. This involves defining, testing and measuring performance and
```

Comment:
It may be worth considering taking an approach similar to SIG Instrumentation, another horizontal SIG with some areas of discrete ownership:
- Own performance and scalability best practices for the project as a whole
- Provide reviews, and guidance to other SIGs
- Own the definition, implementation, and enforcement of scalability SLIs and SLOs for the artifacts produced as part of a Kubernetes release